packages/elements/src/components/card/card-components.tsx

```diff
@@ -60,10 +60,10 @@ export interface CardContextMenuProps extends HTMLAttributes<HTMLDivElement> {
 export interface CardProps extends HTMLAttributes<HTMLDivElement> {
   hasMainCard?: boolean // Should we render an Image card as the main card?
   // Supplied text options for the various fields
-  mainCardHeading?: string
+  mainCardHeading?: string | ReactElement
```

Comment:
`string | ReactElement` better expressed as `ReactNode` as that covers anything that can be rendered eg numbers etc

Reply:
that's the one